src/leap/mail/incoming/service.py

```diff
@@ -246,6 +240,7 @@ def _signal_fetch_to_ui(self, doclist):
         :returns: doclist
         :rtype: iterable
         """
+        # FIXME WTF len(doclist) is 69?
```

Comment:
there was a bug here, the callback was passing a list of strings, instead of the expected list of documents. I've added a commit fixing this.